lib/octokit/error.rb

```diff
@@ -77,7 +77,7 @@ def self.error_for_403(body)
         Octokit::TooManyRequests
       when /login attempts exceeded/i
         Octokit::TooManyLoginAttempts
-      when /returns blobs up to [0-9]+ MB/i
+      when /returns blobs (up to|between) [0-9\-]+ MB/i
```

Comment:
Would you be able to add a test covering the new error message? We have a test for the old error message in `spec/octokit/client_spec.rb:871` which can be pretty much copy-and-pasted.

Reply:
Working on that now, and the regex needs a small modification as well. :)

Reply:
Sorry for the commit spam. I added a test and double checked with regexr.com so I think this is ready to go now. 

Also let me know if you'd rather use a non-capturing group for performance, I went with readability here.